extract.js

```diff
@@ -17,7 +17,8 @@ function safeFeature(feature) {
     return feature;
 }
 
-const features = require('./features');
+const connectionfeatures = require('./connectionfeatures');
```

Comment:
const connectionFeatures = require('./connection-features');

would be more consistent with
const statsMangler = require('./getstats-mangle');